tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/SolutionsControllerGetDashboardTests.cs

```diff
@@ -67,6 +67,17 @@ public async Task ShouldReturnNameId(string id, string name)
             dashboardResult.Id.Should().Be(id);
             dashboardResult.Name.Should().Be(name);
         }
+        
+        [TestCase(null, null)]
+        [TestCase("Sln2", null)]
+        [TestCase(null, "Supplier A")]
+        [TestCase("Sln2", "Supplier A")]
```

Comment:
might as well join the test above with this one, not much point having two separate methods. Just add another parameter on the test case for the ShouldReturnNameId (obviously this should then be renamed)

Reply:
ok